packages/astro/test/parallel.js

```diff
@@ -0,0 +1,24 @@
+import { expect } from 'chai';
+import * as cheerio from 'cheerio';
+import { loadFixture } from './test-utils.js';
+
+describe('Component parallelization', () => {
+	let fixture;
+
+	before(async () => {
+		fixture = await loadFixture({
+			root: './fixtures/parallel/',
+		});
+		await fixture.build();
+	});
+
+	it('renders fast', async () => {
+		let html = await fixture.readFile('/index.html');
+		let $ = cheerio.load(html);
+
+		let firstStart = Number($('.start').first().text());
+		let lastStart = Number($('.start').last().text());
+		let timeTook = lastStart - firstStart;
```

Comment:
All components are kicked off at the same time, so timeTook here will be always be 0 or 1.
![Screenshot (157)](https://github.com/withastro/astro/assets/69170106/d0ddd1e5-e173-49fa-9cf3-fe0c30f86dde)

I think the test should measure the longest time difference instead.
![Screenshot (158)](https://github.com/withastro/astro/assets/69170106/4e252302-6109-42af-beee-10354cb047c0)

```suggestion
		const startTimes  = Array.from($('.start')).map(element => Number(element.children[0].data))
		const finishTimes = Array.from($('.finished')).map(element => Number(element.children[0].data))

		const timeTook = Math.max(...finishTimes) - Math.min(...startTimes);
```

Reply:
For my understanding, the fact that all kick off at the same time is the result of this change. Otherwise, they would render in order and the start times would differ by more than 50ms. The big threshold of 50ms is a bit misleading though (I just copied that from the old pr as well)

Reply:
Yeah I agree the number is misleading. It's what made me think the intention is to measure the render time. Maybe it could reduced and renamed to maybe `kickedOffWithin`?

Reply:
I also liked your idea, so I added the check as well. It's maybe not the cleanest unit test, as it tests more than one thing, but I think it's still OK.